### PR TITLE
CHECK-112-exclude-long-read-fastq-name-check

### PR DESCRIPTION
**Pull request by ottojolanki**

### CODE REVIEW

File: checkfiles.py [hunk 1]
```diff
@@ -323,11 +323,11 @@ def validate_crispr(job, filePath):
 
             except subprocess.CalledProcessError as e:
                 errors['CRISPR_PAM_info_extraction'] = 'Failed to extract information from ' + \
-                                                            local_path
+                                                            filePath
```
Comment (mingjiecn) [inline checkfiles.py:326]: Can we change `filePath` to `file_path`

**Review by mingjiecn — APPROVED** [review]
Should be good if the variable follow the same naming convention

File: checkfiles.py (hunk 1 above)
Comment (ottojolanki) [reply checkfiles.py:326]: I would prefer to touch this code as little as possible, you never know what might call this function and you never know if they use positional or keyword argument style.

Comment (mingjiecn) [reply checkfiles.py:326]: Oh ok!

**Pull request merged**